### DISCUSSION TITLE
使用者友善建議

### DISCUSSION THREAD

**Discussion by Jay413191** [jeecgboot/jimureport#268] [Ideas]
積木報表看似不錯用,但以一個未參與過open source的人來說, 光安裝開發環境就要做很多工, 更別說在打包的過程中可能會出錯,雖有(http://doc.jeecg.com/2043873)文件,但非你族類的很多也聽不懂.建議參考metabase或FINREPORT,就打包好成個jar給人下載, docker image也先包好, 這樣才能擴大使用者用戶,除非你們不打算讓很多人用??但連視頻都做了應該也是想推廣吧??

**Comment by jacobchan** [comment]
点1000个赞
